ARCs/arc-0034.md

```diff
@@ -0,0 +1,63 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of a governance proposal.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The purpose of this ARC is to set the parameters for proposals submitted by the community for evaluation by xGovs prior to a general governance vote for final approval.
+ 
+The Foundation’s goal is to ensure that the xGov process is defined by, and executed by, the Algorand community.
+
+The xGov program will cover evaluation of proposals regarding grants to fund early stage projects and other community input contributions.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+Being able to put measures for a vote is a significant upgrade to the power of the governors. Governors with this power **MUST**:
+- Understand the needs of the community
+- Have their interests aligned with the growth of the Algorand ecosystem
+ 
+### Template to submit a proposal
+
+``` json 
+{
+   "Title":"Name of the proposal",
+   "Project":"Name of the project",
+   "Details":"Details about the proposal",
+   "Category":"Submission Category, DEFI/NFT ...",
+   "Url":"Website url",
+   "Ask":"Grants/ Investment/ Advice",
+   "RoadMap":[
+      {
+         "Miletone":"Explanation of what will be achieved during this period",
+         "Duration":"Duration in month(s) for current milestone"
+      }
+   ]
+}
+```
+
+## Rationale
+In this ARC we seek community input on issues such as:
+- How to submit a proposal?
+- How to distribute xGov funds?
```

Comment:
Each proposal itself should describe how much funding it is asking for and how these funds should be distributed (i.e. delivered at each milestone). Moreover, the proposal itself should also describe if the project could be completed and to what extent with a reduced amount of funding, as well as what is the minimum funding it is asking for.

If the Governors approve together proposals that request a funding total larger than what is available, all should receive proportionally less funding down to their minimum requested amounts.	The rational here being that it is better to give opportunity to many applicants as well as to build things in small steps. If a proposal successfully delivers on the scaled-down plan, it can reapply for further funding later on.

A problem arises if even the minimum requested amounts exceed the available funding. In this case, one option is to automatically reject all the proposals since the Governors should not be allowed to create debt. The proposals would need to be resubmitted and Governors would hopefully make a better overall decision. Another option would be to give priority to proposals asking small amounts of funding, in line with the previous rational of giving reduced funding in case of a great demand. This option would prevent stagnation.